### PR TITLE
[ISSUE-2709,IOTDB-1178] Fix cache not cleared after unseq compaction bug, Fix windows 70,10 ci bug in unseq compaction ci

### PR DESCRIPTION
**Pull request by zhanglingzhe0820**
# ISSUE-2709
## Behavior
User's query operation get invalid chunk after compaction.

## Reason
After compaction, we need to delete a file.
However, the chunk cache and timeseriesMetadata cache is also active which may be queried.
So user can get invalid chunk.

## Solution
We will clear the chunkMetadata cache and timeseriesMetadata cache after a compaction is done.

# IOTDB-1178
## Behavior
A windows ci problem about unseq compaction.
The error behaves like below (The 70,10 problem).
image

## Reason
In unseq compaction, when delete tsfile,  if tsfile is reading now,  will throw exception and then cannot delete the old file in windows system.

## Solution
As we have get the file lock, we just have to clear the TsFileSequenceReader cache before delete a tsfile.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java [hunk 1]
```diff
@@ -155,7 +157,9 @@ private void deleteLevelFilesInList(
   private void deleteLevelFile(TsFileResource seqFile) {
     seqFile.writeLock();
     try {
+      ChunkCache.getInstance().clear();
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java:160]: clear all Cache?

**Comment by HTHou** [comment]
Don't forget merge master branch....

**Review by JackieTien97 — APPROVED** [review]

File: example/udf/pom.xml [hunk 2]
```diff
@@ -73,7 +73,7 @@
                         <importOrder>
                             <order>org.apache.iotdb,,javax,java,\#</order>
                         </importOrder>
-                        <removeUnusedImports/>
```
Comment (HTHou) [inline example/udf/pom.xml:76]: Don't change this...

File: pom.xml [hunk 3]
```diff
@@ -559,7 +559,6 @@
                             <exclude>**/.git/**</exclude>
                             <exclude>**/.mvn/**</exclude>
                             <exclude>**/.gitignore</exclude>
-                            <exclude>**/.git-blame-ignore-revs</exclude>
```
Comment (HTHou) [inline pom.xml:562]: Same

File: example/udf/pom.xml [hunk 4]
```diff
@@ -73,7 +73,7 @@
...
+                        <removeUnusedImports />
```
Comment (HTHou) [inline example/udf/pom.xml:76]:
```suggestion
                        <removeUnusedImports/>
```

File: pom.xml [hunk 5]
```diff
@@ -559,7 +559,6 @@
                             <exclude>**/.git/**</exclude>
                             <exclude>**/.mvn/**</exclude>
                             <exclude>**/.gitignore</exclude>
```
Comment (HTHou) [inline pom.xml:561]:
```suggestion
                            <exclude>**/.gitignore</exclude>
                            <exclude>**/.git-blame-ignore-revs</exclude>
```

File: pom.xml [hunk 6]
```diff
@@ -613,7 +612,7 @@
                             <importOrder>
                                 <order>org.apache.iotdb,,javax,java,\#</order>
                             </importOrder>
-                            <removeUnusedImports/>
+                            <removeUnusedImports />
```
Comment (HTHou) [inline pom.xml:615]:
```suggestion
                            <removeUnusedImports/>
```

**Comment by HTHou** [comment]
Need to merge master branch again..

**Comment by zhanglingzhe0820** [comment]
cherry-pick to 0.11 [PR](https://github.com/apache/iotdb/pull/2749)

**Pull request merged**